cmds/report.c

```diff
@@ -184,6 +184,39 @@ static void print_and_delete(struct rb_root *root, bool sorted, void *arg,
 	}
 }
 
+static void print_csv_function(struct uftrace_report_node *node, void *unused)
+{
+        if (avg_mode == AVG_NONE) {
+                print_time_unit(node->total.sum);
```

Comment:
We need a special care about `print_time_unit` as it already prints the time unit with its internal formating. I would like to ask you to provide a flag to `print_time_unit` whether it uses formating internal or not.

This change is better to be in a separate commit before making CSV related change.

Reply:
@namhyung Do you agree adding a new flag to `print_time_unit`?
```diff
-void print_time_unit(uint64_t delta_nsec)
+void print_time_unit(uint64_t delta_nsec, bool needs_align)
 {
-        __print_time_unit(delta_nsec, false);
+        __print_time_unit(delta_nsec, false, needs_align);
 }
```